tests/unit/test_network_of_populations.py

```diff
@@ -6,7 +6,7 @@
 import networkx as nx
 import pytest
 
-from simple_network_sim import network_of_populations as np, loaders
```

Comment:
What has prompted this change, please?

Reply:
I think the line below this is `from simple_network_sim import network_of_populations as np`. `loaders` was not used in the file, so I removed it (it was my IDE's idea)